runtime-sdk/modules/rofl-market/src/types.rs

```diff
@@ -411,6 +411,17 @@ pub struct InstanceCreate {
     pub term_count: u64,
 }
 
+/// A request by the existing instance admin to change the admin.
+#[derive(Clone, Debug, Default, cbor::Encode, cbor::Decode)]
+pub struct InstanceTransferAdmin {
```

Comment:
Would name `InstanceChangeAdmin` be more appropriate, as you are changing admin, but transferring admin rights?

Reply:
Yeah good point, will rename the method.